app/assets/stylesheets/shared.css.scss

```diff
@@ -60,10 +60,31 @@ legend
   text-align: left;
   vertical-align: middle;
 }
+
 .modal-title{
   font-size: 16px;
   font-weight: bold;
 }
+
 .modal-header{
   padding-bottom: 7px;
+}
+
+.badge {
+  display: inline-block;
+  min-width: 10px;
+  padding: 3px 7px;
+  font-size: 12px;
+  font-weight: bold;
+  color: white;
+  line-height: 1;
+  vertical-align: baseline;
+  white-space: nowrap;
+  text-align: center;
+  background-color: #9F0606;
+  border-radius: 50%;
```

Comment:
Since border-radius is used, were other four border-radius attributes necessary?